Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -444,4 +444,4 @@ public void AfterTest(ITest test)
 
         public ActionTargets Targets => ActionTargets.Test | ActionTargets.Suite;
     }
-}

```

Comment:
actually we leave 1 new line at the end of the file for purpose so I would prefer not to delete it

Reply:
Fixed